Makefile

```diff
@@ -49,10 +49,15 @@ ifeq ($(HAS_GO), yes)
 	CGO_CFLAGS ?= $(shell $(GO) env CGO_CFLAGS) $(CGO_EXTRA_CFLAGS)
 endif
 
-ifeq ($(OS), Windows_NT)
-	GOFLAGS := -v -buildmode=exe
-	EXECUTABLE ?= gitea.exe
-else ifeq ($(OS), Windows)
+ifeq (Windows, $(findstring Windows,$(OS)))
+	ifeq (,$(GOOS))
+		IS_WINDOWS := yes
+	endif
+	ifeq (windows,$(GOOS))
```

Comment:
```suggestion
	ifeq ($(GOOS),)
		IS_WINDOWS := yes
	endif
	ifeq ($(GOOS),windows)
```

No yoda conditions.

Reply:
Thanks, updated

(didn't know it was called Yoda conditions 😄 )

Reply:
https://en.wikipedia.org/wiki/Yoda_conditions 😆